common/persistence/persistence-tests/metadataPersistenceV2Test.go

```diff
@@ -170,6 +170,7 @@ func (m *MetadataPersistenceSuiteV2) TestCreateDomain() {
 	m.Nil(resp1.FailoverEndTime)
 	m.Equal(lastUpdateTime, resp1.LastUpdatedTime)
 
+	var isolationgroups types.IsolationGroupConfiguration
```

Comment:
Could you move this to the first `CreateDomain` call in this test?

Reply:
Could you update the `TestGetDomain` to check the field of `isolationGroupConfiguration`?